src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+
+        System.out.println("Truck machine started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+
+        System.out.println("Truck machines stopped working");

```

Comment:
Similarly, the message in the `stopWork` method should be more informative. It should indicate that it is a `Truck` that has stopped working. For instance: 'Truck has stopped working.'